articles/search/search-indexer-field-mappings.md

```diff
@@ -378,6 +378,29 @@ When errors occur that are related to document key length exceeding 1024 charact
 ]
  ```
 
+<a name="toJsonFunction"></a>
+
+### toJson function
+ 
+This function converts a string into a formatted JSON object. This can be used for scenarios where the data source doesn't natively support compound / hierarchical data types, such as Azure SQL and then map it to complex fields.
```

Comment:
line 385 can you replace / with or "compound or hierarchical data types" Also, "such as Azure SQL" needs a comma, and it needs to relocate near "data source" otherwise, it looks like it's qualifying data type, not data source.